scripts/after_install.sh

```diff
@@ -45,14 +45,15 @@ psql -U postgres -d just-spaces -c "CREATE EXTENSION IF NOT EXISTS postgis"
 
 # OPTIONAL Run migrations and other management commands that should be run with
 # every deployment
-$VENV_DIR/bin/python $PROJECT_DIR/manage.py migrate
-$VENV_DIR/bin/python $PROJECT_DIR/manage.py createcachetable
-$VENV_DIR/bin/python $PROJECT_DIR/manage.py collectstatic --no-input
+sudo -H -u datamade $VENV_DIR/bin/python $PROJECT_DIR/manage.py migrate
+sudo -H -u datamade $VENV_DIR/bin/python $PROJECT_DIR/manage.py createcachetable
+sudo -H -u datamade $VENV_DIR/bin/python $PROJECT_DIR/manage.py collectstatic --no-input
 
 # Echo a simple nginx configuration into the correct place, and tell
 # certbot to request a cert if one does not already exist.
 # Wondering about the DOMAIN variable? It becomes available by source-ing
 # the config file (see above).
+sudo apt install python-certbot-nginx
```

Comment:
Does this need to be added to our deployment templates?

Reply:
I think this should be added to the server configs in this line: https://github.com/datamade/deploy-a-site/blob/cbacb18a0b8f612a76b7e4877576cb4e124e8a86/templates/server/%7B%7Bcookiecutter.directory_name%7D%7D/scripts/after_install.sh#L16

Does that seem right to you?

Reply:
That's the right place, but it looks like it's on there already:

```
packages='apt-transport-https ca-certificates curl software-properties-common nginx supervisor python3 python3-pip python3-venv certbot python-certbot-nginx'
```

Any idea why it wasn't working for you?

Reply:
Because I copy and pasted from here instead!! https://github.com/datamade/deploy-a-site/blob/master/Basic-server-setup.md

I can put together a PR on that repo to reflect that change

Reply:
Ah, gotcha!